src/main/scala/mesosphere/marathon/storage/migration/MigrationTo15.scala

```diff
@@ -58,14 +58,14 @@ private[migration] object MigrationTo15 {
     val mbn = mesosBridgeName
     // lazily evaluate the special environment variable and configured network name: we might never need them, and in
     // that case we don't want to abort migration (because there's no reason to).
-    AppsResource.appNormalization(AppsResource.NormalizationConfig(
+    AppHelpers.appNormalization(
```

Comment:
I love the reorganization of functions to AppHelper!

Reply:
Kudos to @timcharper!